pkg/etcdutil/types.go

```diff
@@ -25,6 +25,8 @@ const (
 
 	// DefaultDefragConnectionTimeout for timeout during ETCD defrag call
 	DefaultDefragConnectionTimeout time.Duration = 30 * time.Second
+	// DefaultSnapshotTimeout defines default timeout for taking FullSnapshot.
+	DefaultSnapshotTimeout time.Duration = 8 * time.Minute
```

Comment:
Any particular reason to set the default value to `8 minutes`?

Reply:
Amshu suggested that we could technically have an 8GB snapshot, so just to be on the safe side.
https://github.com/gardener/etcd-backup-restore/pull/361#pullrequestreview-718174803